snippets/article-card.liquid

```diff
@@ -16,7 +17,7 @@
   <a href="{{ article.url }}" class="article-content motion-reduce">
     {%- if show_image == true and article.image -%}
       <div class="article-card__image-wrapper">
-        <div class="article-card__image media media--landscape">
+        <div class="article-card__image media {% if image_height %}article-card__image--{{ image_height }}{% else %}media--landscape{% endif %}" {% if section.settings.image_height == 'adapt' %} style="padding-bottom: {{ 1 | divided_by: article.image.aspect_ratio | times: 100 }}%;"{% endif %}>
           <img
             srcset="{%- if article.image.src.width >= 165 -%}{{ article.image.src | img_url: '165x' }} 165w,{%- endif -%}
               {%- if article.image.src.width >= 360 -%}{{ article.image.src | img_url: '360x' }} 360w,{%- endif -%}
```

Comment:
Below in this file, where it shows the date and the author, we might run into this layout since we can now have 4 posts: ![](https://screenshot.click/21-09-9we5t-2cbmn.png)

It was probably possible before but more likely now it's a bit tighter.

Reply:
Ahh yes, forgot about the author wrap. We can't possibly guarantee that wont ever wrap regardless, so I'm inclined to say adding the option to display a 4th post is worth the increased chances of this happening. But design may feel more strongly about it @Oliviammarcello

Reply:
Sounds good to me 👍  And I agree that we can't prevent it from happening either way.